src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_References/Main.java

```diff
@@ -0,0 +1,35 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_References;
+
+public class Main {
+    public static void main(String[] args) {
+
+// Arrays_Ex2
+//        int[] array = Arrays_Ex2.createArray(5); // assigns the array created in the method createArray and attributes its length (5)/ {1,2,3,4,5}
```

Comment:
Uncomment for testing methods or delete them

Reply:
Done